templates/data/_form-data-valkey.html

```diff
@@ -0,0 +1,174 @@
+<div class="p-modal" style="display: none;" id="data-valkey-modal">
+  <div class="p-modal__dialog" role="dialog">
+    <div class="js-pagination js-pagination--1">
+      <div class="row--50-50 p-strip is-shallow">
+        <div class="col">
+          <h2 class="p-heading--1">Talk to our Valkey experts</h2>
+        </div>
+        <div class="col">
+          <button class="p-modal__close p-modal-close-button"
+                  aria-controls="data-valkey-modal"
+                  aria-label="Close active modal"
+                  style="margin-top: 0">Close</button>
+          <form action="https://ubuntu.com/marketo/submit"
+                method="post"
+                id="mktoForm_5966"
+                class="modal-form p-form p-form--stacked">
+            <h3 class="p-heading--5">How should we get in touch?</h3>
+            <div class="p-section--shallow p-form-validation">
+              <label for="firstName" class="is-required">First name:</label>
+              <input required
+                     id="firstName"
+                     name="firstName"
+                     maxlength="255"
+                     type="text"
+                     required="required" />
+              <label for="lastName" class="is-required">Last name:</label>
+              <input required
+                     id="lastName"
+                     name="lastName"
+                     maxlength="255"
+                     type="text"
+                     required="required" />
+              <label for="email" class="is-required">Work email:</label>
+              <input required
+                     id="email"
+                     name="email"
+                     maxlength="255"
+                     type="email"
+                     pattern="^[^ ]+@[^ ]+\.[a-z]{2,26}$"
+                     required="required" />
+              {% with required="false", remove_list=True %}
+                {% include "shared/forms/_country.html" %}
+              {% endwith %}
+              <div class="p-section--shallow">
+                <label for="phone">Phone number:</label>
+                <input id="phone" name="phone" maxlength="255" type="tel" />
+              </div>
+              <label for="company">Company:</label>
+              <input id="company" name="company" maxlength="255" type="text" />
+              <label for="title">Job title:</label>
+              <input id="title" name="title" maxlength="255" type="text" />
+            </div>
+            <h3 class="p-heading--5">Tell us more about your Spark use case</h3>
+            <textarea id="Comments_from_lead__c"
+                      name="Comments_from_lead__c"
+                      aria-label="Anything you'd like to communicate about your needs or interests?"
+                      rows="3"
+                      placeholder="Anything you'd like to communicate about your needs or interests?"></textarea>
```

Comment:
I'm not sure about this placeholder text. Imo it doesn't comply with the title of the field which is asking about a Valkey use case. But the placeholder is asking about anything you would like to discuss. @eliman11 Are you ok with this?

Reply:
This was copied over from an existing form, but I can also remove it if that's better? If new placeholder texts requires going back and forth with marketing then I don't think we have time for that

Reply:
This text is not specified in the copy doc. Was it specified in the existing form's copy doc?

Reply:
I'm going to go ahead and guess no because placeholder is almost never specified in the copy doc. @eliman11 please let me know which of the 2 options you prefer, amend or remove?

Reply:
Please move that field to the top for consistency with the other forms in the bubble, I applied the change to the [copy doc](https://docs.google.com/document/d/1SZ3vEvBow6dBOSzU97uwzgdiMI8_hHbW_Q_RFgJnwJY/edit). Don't get rid of the placeholder yet, it's fine and it's present in all other forms so I'd prioritise consistency for now: @Sophie-32 please evaluate if we should get rid of it here and elsewhere. @mtruj013 feel free to change label to UX+1 once you address this.